src/utils/daos_metrics/daos_metrics.c

```diff
@@ -52,9 +52,10 @@ print_usage(const char *prog_name)
 	       "--gauge, -g\n"
 	       "\tInclude gauges\n"
 	       "--read, -r\n"
-	       "--reset, -e\n"
-	       "\tInclude timestamp of when metric was read\n",
-	       prog_name);
+               "\tInclude timestamp of when metric was read\n"
+               "--reset, -e\n"
+               "\tReset all metrics to zero\n",
+ 		prog_name);
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs